packages/framework/src/Resources/views/Common/robots.txt.twig

```diff
@@ -1,2 +1,6 @@
+{% if customContent is not empty %}
+{{ customContent }}
+
+{% endif %}
```

Comment:
```suggestion
{% if customContent is not empty %}
    {{ customContent }}
{% endif %}
```

Reply:
It is formatted like that because it's need for having resulting file formatted correctly.

```
Disallow: /admin

Sitemap: http://127.0.0.1:8000/content/sitemaps/domain_1_sitemap.xml
Image Sitemap: http://127.0.0.1:8000/content/sitemaps/domain_1_sitemap_image.xml
```

vs 

```
    Disallow: /admin
Sitemap: http://127.0.0.1:8000/content/sitemaps/domain_1_sitemap.xml
Image Sitemap: http://127.0.0.1:8000/content/sitemaps/domain_1_sitemap_image.xml
```

But I found out that it can be written a little bit better, is it OK for you?

```twig
{%- if customContent is not empty -%}
    {{ customContent }}

{% endif %}
```

Reply:
I think you should use `~` instead of `-`